src/Base/Console.h

```diff
@@ -471,7 +471,18 @@ enum class LogStyle{
     Log,
     Critical,               // Special message to mark critical notifications
     Notification,           // Special message for notifications to the user (e.g. educational)
-    TranslatedNotification, // Special message for already translated notifications to the user (e.g. educational)
+};
+
+enum class IntendedRecipient {
+    All,                    // All recipients, Developers and Users (One notification covers all)
+    User,                   // User only (notification intended only for a user)
+    Developer,              // Developer only (notification intended only for a developer)
+};
+
+enum class ContentType {

```

Comment:
I really like this design (thanks for the strong typing!) -- can you add a bit of documentation here explaining how a developer should use this? e.g.
```
/// Use the QT_TRANSLATE_NOOP("Notification", "Static error string here") when using `Untranslated`
/// Use tr() when using Translated so that the string is pre-translated.
```
(or something like that)

Reply:
Thank you. Sure!